beacon_node/http_api/src/attestation_rewards.rs

```diff
@@ -0,0 +1,260 @@
+use beacon_chain::{BeaconChain, BeaconChainTypes};
```

Comment:
Ah, just realised this file is in `http_api`. I think for consistency we should move it to `beacon_node/beacon_chain/src/attestation_rewards.rs`. Let me know if you need help with the refactor